core/src/main/java/de/evoila/cf/broker/service/DeploymentService.java

```diff
@@ -18,16 +18,43 @@
  */
 public abstract interface DeploymentService {
 	
-
+	/**
+	 * @return All known ServiceInstances
+	 */
+	// List<ServiceInstance> getAllServiceInstances();
+	/**
+	 * @param id
+	 * @return The ServiceInstance with the given id or null if one does not
+	 *         exist
```

Comment:
If Code is commented out, please remove it. Therefore we have git

Reply:
Its not code. Its not a comment.